angular-ngc/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "lib": ["es2020", "dom"],
+    "paths": {
+      "@ngc-example/*": ["packages/*"]
+    }
+  },
+  "angularCompilerOptions": {
+    "compilationMode": "full",
```

Comment:
in a follow-up, could you add more comments in any places that are load-bearing for making the whole thing work?

Reply:
I've added some comments and dropped options which aren't necessary for the example.